staging/src/k8s.io/kubectl/pkg/cmd/delete/delete.go

```diff
@@ -294,20 +299,28 @@ func (o *DeleteOptions) RunDelete(f cmdutil.Factory) error {
 }
 
 func (o *DeleteOptions) DeleteResult(r *resource.Result) error {
-	found := 0
 	if o.IgnoreNotFound {
 		r = r.IgnoreErrors(errors.IsNotFound)
 	}
+
+	deletedInfos, err := r.Infos()
```

Comment:
naming nit: `deletedInfos` is misleading now, since they definitely haven't been deleted at this point.